code/game/dna/genes/vg_powers.dm

```diff
@@ -119,4 +119,31 @@ Obviously, requires DNA2.
 	// Can't be big AND small.
 	if((M.sdisabilities & BLIND) || (M.disabilities & NEARSIGHTED))
 		return 0
-	return ..(M,flags)
\ No newline at end of file
+	return ..(M,flags)
+
+var/global/list/NOIRLIST = list(0.3,0.3,0.3,0,\
+					 			0.3,0.3,0.3,0,\
+								0.3,0.3,0.3,0,\
+								0.0,0.0,0.0,1,)
+
+/datum/dna/gene/basic/noir
+	name="Noir"
+	desc = "In recent years, there's been a real push towards 'Detective Noir' movies, but since the last black and white camera was lost many centuries ago, Scientists had to develop a way to turn any movie noir."
+	activation_messages=list("The Station's bright coloured light hits your eyes for the last time, and fades into a more appropriate tone, something's different about this place, but you can't put your finger on it. You feel a need to check out the bar, maybe get to the bottom of what's going on in this godforsaken place.")
+	deactivation_messages = list("You now feel soft boiled.")
+
+	mutation=M_NOIR
+
+/datum/dna/gene/basic/noir/New()
+	block=NOIRBLOCK
+	..()
+
+/datum/dna/gene/basic/noir/activate(var/mob/M)
+	..()
+	if(M.client)
+		M.client.color = NOIRLIST
+
+/datum/dna/gene/basic/noir/deactivate(var/mob/M,var/connected,var/flags)
+	if(..())
+		if(M.client && M.client.color == NOIRLIST)
```

Comment:
What happens if I log out when this is removed?

What happens if I log out and have the gene currently?